components/resources/rewards_strings.grdp

```diff
@@ -818,4 +818,65 @@
     Your BAT earnings will be paid to this payout account.
   </message>
 
+  <message name="REWARDS_AC_AMOUNT_LABEL" desc="">
+    You will send
+  </message>
+  <message name="REWARDS_AC_AMOUNT_TEXT" desc="">
+    Up to <ph name="AMOUNT">$1</ph> BAT
+  </message>
+  <message name="REWARDS_AC_ATTENTION_LABEL" desc="">
+    Attention
+  </message>
+  <message name="REWARDS_AC_DISABLED_TEXT" desc="">
+    An automatic way to support websites and content creators, based on how much time you spend on their content.
+  </message>
+  <message name="REWARDS_AC_DISABLED_TITLE" desc="">
+    Auto-Contribute is disabled
+  </message>
+  <message name="REWARDS_AC_EMPTY_LIST_TEXT" desc="">
+    Verified Creators that you visit will appear here as you browse.
```

Comment:
Nit: I think we only capitalize "creators" when it's "Brave Creators" now.

Reply:
Ah yes, let's make it lowercase: `Verified creators that you visit will appear here as you browse.`

Reply:
@zenparsing I think the current pre3.0 UI, there are instances of "Verified Creator" (as in when it shows verification status in the UI). We can change all of those specific instances to Verified creator" too.

Reply:
Done